package.json

```diff
@@ -37,7 +37,10 @@
     "clean": "git clean -Xdf",
     "release": "np",
     "deploy-docs": "cd docs && yarn install && yarn deploy",
-    "postpublish": "yarn deploy-docs"
+    "postpublish": "yarn deploy-docs",
+    "pretty": "prettier --write \"./**/*.{js,json,md}\"",
```

Comment:
This automatically gets run on commit. Do we need a script for running it manually?

Reply:
Yeah, wasn't too sure if these were needed or not, I like having them to run manually because sometimes stuff slips through the cracks.